include/ruby/internal/intern/load.h

```diff
@@ -176,6 +176,19 @@ VALUE rb_f_require(VALUE self, VALUE feature);
  */
 VALUE rb_require_string(VALUE feature);
 
+/**
+ * Resolves and returns a symbol in the native extension specified by the
+ * feature and symbol names. Extensions will use this function to access the
+ * symbols provided by other native extensions.
+ *
+ * @param[in]  feature           Name of a feature, e.g. `"json"`.
+ * @param[in]  symbol            Name of a symbol defined by the feature.
+ * @return     The resolved symbol, defined and externed by the specified
+ *             feature. It may be NULL when the feature is not loaded, the
+ *             feature was not extension, or the symbol was not found.
```

Comment:
Actually, the current implementation raises LoadError when the feature exists but the symbol is not found.
Should it suppress the LoadError?

Reply:
It is questionable to me.

Reply:
Finally, I think that returning NULL in both cases (the feature not found and the symbol not found) should be consistent.